homeassistant/components/sonos/media_player.py

```diff
@@ -621,9 +624,9 @@ def update_media_radio(self, variables, track_info):
         media_info = self.soco.avTransport.GetMediaInfo([("InstanceID", 0)])
         self._media_image_url = self._radio_artwork(media_info["CurrentURI"])
 
-        self._media_artist = track_info.get("artist")
+        self._media_artist = track_info.get("artist") or None
```

Comment:
Maybe it would be better to add these `or None` to the property getters for less duplication. We already do that for `media_image_url`.

Reply:
It seems like this change made the PR miss its target because the state property reads from `self._media_title` which is no longer set to `None`.

I have included a proposal to fix that in #34311 (by using `self.media_title`).